login/index.html

```diff
@@ -7,50 +7,56 @@
   <link rel="stylesheet" href="/css/global.css">
   <link rel="stylesheet" href="/css/login.css">
   <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css" />
-  <title>판다마켓</title>
+  <script src="/src/login.js" type="module"></script>
+  <title>판다마켓 로그인</title>
 </head>
 <body>
   <main class="login__main">
     <div class="login-container">
       <section class="login-logo">
         <a href="/" title="판다마켓 홈페이지 바로가기">
-          <img src="/images/logo_lg.png" alt="판다마켓 로고" class="login-logo__img">
+          <img src="/images/logo_lg.png" alt="판다마켓 로고" class="login-logo-img">
         </a>
       </section>
       <section class="form-container">
         <form action="" method="get" class="form-login">
-          <div class="form__input-container">
+          <div class="form-input-container">
             <label for="email">이메일</label>
-            <input id="email" type="email" name="email" class="form__input" placeholder="이메일을 입력해주세요" />
+            <input id="email" type="email" name="email" class="form-input" placeholder="이메일을 입력해주세요" />
+            <span class="input-validation-error" style="visibility: hidden;"></span>
           </div>
-          <div class="form__input-container">
+          <div class="form-input-container">
             <label for="password">비밀번호</label>
             <div class="password-container">
-              <input id="password" type="password" name="password" class="form__input" placeholder="비밀번호를 입력해주세요" />
-              <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" class="visible-icon">
+              <input id="password" type="password" name="password" class="form-input" placeholder="비밀번호를 입력해주세요" />
+              <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" class="invisible-icon" aria-label="비밀번호 보기">
```

Comment:
P2:
`aria-label`은 상호작용이 가능한 요소에 어떤 요소인지 설명하는 속성입니다. 
해당 요소는 상호작용이 가능하니 아래처럼 하면 더 좋을 것 같습니다.

```suggestion
              // button 상태에 따라 JS로 aria-label을 바꿔주면 좋을 것 같습니다.
              <button type='button' "invisible-icon" aria-label="비밀번호 보기">
                  <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" aria-description='눈 감은 아이콘'> ... </svg>
                  <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" aria-description='눈 뜬 아이콘'> ... </svg>
              </button>
```